client/internal/v2/client_test.go

```diff
@@ -499,7 +499,7 @@ func (f fakeCancelContext) Done() <-chan struct{} {
 func (f fakeCancelContext) Err() error        { return errFakeCancelContext }
 func (f fakeCancelContext) Value(key any) any { return 1 }
 
-func withTimeout(parent context.Context, timeout time.Duration) (
+func withTimeout(parent context.Context, _timeout time.Duration) (
```

Comment:
a quick question, why add a prefix "_" for the parameter? Is it required by the linter?

Reply:
If the function has unused argument, the lint will blame it. Using `_X` is to ignore the lint error and it can be used in the future.